setup.cfg

```diff
@@ -1,2 +1,2 @@
 [pep8]
-exclude = ./venv,./bower_components,./digital_marketplace_frontend_toolkit
```

Comment:
Why does this also remove `./digital_marketplace_frontend_toolkit`? It doesn't mention it in the commit comment.